### PR TITLE
Added throttling reasons and cpu uptime

### PR DESCRIPTION
**Pull request by gerth2**
While doing some debugging last night, I had a number of issues with my power supply on the Pi. When I hooked up a keyboard and screen manually I quickly found root cause due to the messages in the console from the kernel, but I wasn't seeing these show up anywhere else in an obvious fashion.

The symptoms actually align some ancedotal reports from @dkt01 of settings not saving and recalling - if a reboot had happened on or during the settings write, it could cause intermittent issues.

I added Raspberry-Pi specific throttling reason and CPU uptime metrics to the Settings/General tab to help identify this as root cause.

image

### CODE REVIEW

File: photon-client/src/views/SettingsViews/General.vue [hunk 1]
```diff
@@ -49,22 +49,70 @@
...
+                    LV: Low Supply Voltage <br>
+                    FRQ: ARM Core Frequency is Capped<br>
+                    THR: CPU Throttling Active<br>
+                    TEMP: CPU Over-Temperature<br>
+              </span>
+            </v-tooltip>
+          </th>
+          <th class="infoElem">
+            CPU Uptime
+          </th>
```
Comment (mcm001) [inline photon-client/src/views/SettingsViews/General.vue:90]: Should this only appear on Pi platforms?

File: photon-client/src/views/SettingsViews/General.vue [hunk 2]
```diff
@@ -49,22 +49,70 @@
...
+                  v-on="on"
+                >
+                ⓘ CPU Prev Throttling 
+                </span>
+              </template>
+              <span>
+                    LV: Low Supply Voltage <br>
+                    FRQ: ARM Core Frequency is Capped<br>
+                    THR: CPU Throttling Active<br>
+                    TEMP: CPU Over-Temperature<br>
```
Comment (mcm001) [inline photon-client/src/views/SettingsViews/General.vue:84]: Are these messages generic enough to apply to non-pi linux platforms?

Comment (gerth2) [reply photon-client/src/views/SettingsViews/General.vue:84]: They are not - the reason codes are currently PI-specific

**Review by mcm001 — APPROVED** [review]

**Review by mcm001 — CHANGES_REQUESTED** [review]
Wait why is index.html no longer the default UI has not been copied text?

**Comment by mcm001** [comment]
I'm bad, it wasn't broken by this pr -- we should fix but I'll merge regardless

**Review by mcm001 — APPROVED** [review]

**Pull request merged**